koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -570,22 +516,25 @@ def sigint_handler(*args, **kwargs):
         # so we have to collect them back when the process ends
         user_warnings = self._read_user_warnings(osbs_logs_dir)
 
-        response = self.osbs().wait_for_build_to_finish(build_id)
```

Comment:
Do we not wait for the build to finish anymore, or does something else do that?

Reply:
reading logs is waiting and following logs:

             logs = self.osbs().get_build_logs(build_id, follow=True, wait=True)